assets/js/app.js

```diff
@@ -0,0 +1,94 @@
+/* window.onscroll = function () {
+  if (document.body.scrollTop > 0 || document.documentElement.scrollTop > 0) {
+    document.getElementById('navbar').classList.add('scrolled');
+  } else {
+    document.getElementById('navbar').classList.remove('scrolled');
+  }
+}; */
+
+/* AOS.init({
+  duration: 800,
+}); */
```

Comment:
Please, do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.